data/src/main/java/ca/bc/gov/data/datasource/local/dao/TestResultDao.kt

```diff
@@ -32,6 +32,6 @@ interface TestResultDao {
     @Query("SELECT id FROM test_result WHERE patient_id = :patientId AND collection_date = :collectionDate")
     suspend fun getTestResultId(patientId: Long, collectionDate: Instant): Long?
 
-    @Query("DELETE FROM test_result WHERE patient_id = :patientId AND data_source = 'BCSC'")
+    @Query("DELETE FROM test_result WHERE patient_id = :patientId")
     suspend fun deleteAuthenticatedTestRecords(patientId: Long): Int
```

Comment:
rename method to delete()

Reply:
renamed to deleteTestRecords() as delete() already exists